ngraph_bridge/ngraph_builder.cc

```diff
@@ -2945,27 +2835,16 @@ static Status TranslateNonMaxSuppressionV4Op(
         score_threshold.size());
   }
 
-  std::string backend_name;
-  TF_RETURN_IF_ERROR(ngraph_bridge::GetNodeBackend(op, &backend_name));
+  auto& ng_max_output_size = ng::opset3::Constant::create(element::i64, Shape{}, max_output_size[0]);
+  auto& ng_iou_threshold = ng::opset3::Constant::create(element::f32, Shape{}, iou_threshold[0]);
+  auto& ng_score_threshold = ng::opset3::Constant::create(element::f32, Shape{}, score_threshold[0]);
 
-  auto config_map = BackendManager::GetBackendAttributeValues(backend_name);
-  if (config_map.at("ngraph_backend") != "NNPI") {
-    return errors::Internal("In translating NonMaxSuppressionV4 op ",
-                            op->name(), " found requested backend ",
-                            backend_name, " which is unsupported");
-  }
+  shared_ptr<ng::Node> ng_nmsv4 =
+      ConstructNgNode<ng::opset3::NonMaxSuppression>(
+          op->name(), ng_boxes, ng_scores, ng_max_output_size, ng_iou_threshold,
+          ng_score_threshold);
 
-  ng::runtime::Backend* backend = BackendManager::GetBackend(backend_name);
-
-  shared_ptr<ng::Node> ng_nmsv4 = backend->get_backend_op(
-      "NonMaxSuppressionV4", &ng_boxes, &ng_scores,
-      (size_t)(max_output_size[0]), (float)(iou_threshold[0]),
-      (float)score_threshold[0], (bool)pad_to_max_output_size);
-  if (ng_nmsv4 == nullptr) {
-    return errors::Internal("In translating NonMaxSuppressionV4 op ",
-                            op->name(),
-                            " backend could not return valid ngraph node");
-  }
+  // ADK: (FIXME) are the outputs set correctly here?
```

Comment:
Do we have tests for this op? I couldn't find any, so we should probably add one.

Reply:
None of the existing backends support this op. And I believe that there are TF unit tests that test this op which we'll enable once we've a backend that supports this op.

Reply:
With this PR we have a translation that is not tested. Better to add support when it could be tested.

Reply:
I agree, we should probably only remove the backend specific ops in this PR and add/change support if we can test it.

Reply:
`NonMaxSuppressionV4` is not a backend-specific op. I agree that it's untested at the moment but so are several other ops. I'd rather keep it and fix it as soon as the new backend is up (which is going to be very soon) than comment it out temporarily.